spec/isec2_linux.go

```diff
@@ -0,0 +1,41 @@
+package spec
+
+import (
+	"io/ioutil"
+	"strings"
+	"time"
+
+	"github.com/Songmu/retry"
+)
+
+// If the OS is Linux, check /sys/hypervisor/uuid file first. If UUID seems to be EC2-ish, call the metadata API (up to 3 times).
+// ref. http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/identify_ec2_instances.html
+func isEC2() bool {
+	data, err := ioutil.ReadFile("/sys/hypervisor/uuid")
+	if err != nil {
+		// Probably not EC2.
+		return false
+	}
+	// Probably not EC2.
```

Comment:
What does `Probably` mean? Do you mean this method can return false even if in EC2?